velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -419,12 +405,55 @@ void Writer::setMemoryReclaimers() {
   generalPool_->setReclaimer(exec::MemoryReclaimer::create());
 }
 
+namespace {
+
+std::optional<TimestampUnit> getTimestampUnit(
+    const Config& config,
+    const char* configKey) {
+  if (const auto unit = config.get<uint8_t>(configKey)) {
+    VELOX_CHECK(
+        unit == 0 /*second*/ || unit == 3 /*milli*/ || unit == 6 /*micro*/ ||
+            unit == 9 /*nano*/,
+        "Invalid timestamp unit: {}",
+        unit.value());
+    return std::optional(static_cast<TimestampUnit>(unit.value()));
+  }
+  return std::nullopt;
+}
+
+} // namespace
+
+void WriterOptions::processSessionConfigs(const Config& config) {
+  if (!parquetWriteTimestampUnit) {
+    parquetWriteTimestampUnit =
+        getTimestampUnit(config, "hive.parquet.writer.timestamp_unit");
```

Comment:
Can we have "hive.parquet.writer.timestamp_unit" and "hive.parquet.writer.timestamp-unit" as `static constexpr const char*` in parquet `WriterOptions`?